sdk-core/src/main/java/care/data4life/sdk/ApiService.java

```diff
@@ -241,7 +241,8 @@ Completable uploadTagEncryptionKey(String alias, String userId, String encrypted
         return service.createRecord(alias, userId, (EncryptedRecord) encryptedRecord);
     }
 
-    Observable<List<EncryptedRecord>> fetchRecords(String alias,
+    // TODO remove public
+    public Observable<List<EncryptedRecord>> fetchRecords(String alias,
```

Comment:
What's this about? Does public means it will be visible on the client apis like in swift?

Reply:
Since the new Service lives not in the same package but needs access I had to change the visibility. `public` means it is [visible](https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html) to all other classes.